src/mscorlib/model.xml

```diff
@@ -6202,15 +6202,15 @@
       <Member Name="StringToHGlobalUni(System.String)" />
       <Member Name="StructureToPtr&lt;T&gt;(T,System.IntPtr,System.Boolean)" />
       <Member Name="UnsafeAddrOfPinnedArrayElement&lt;T&gt;(T[],System.Int32)" />
-      <Member Name="WriteByte(System.Object,System.Int32,System.Byte)" />
+      <Member Name="WriteByte(System.Object,System.Int32,System.Byte)" Condition="FEATURE_LEGACYSURFACE"/>
```

Comment:
It would be nice to move all Write*(System.Object methods together like it is done for the Read methods.